light/rpc/client.go

```diff
@@ -420,15 +420,29 @@ func (c *Client) Validators(height *int64, page, perPage *int) (*ctypes.ResultVa
 		return nil, errNegOrZeroHeight
 	}
 
+	updateHeight := res.BlockHeight - 1
```

Comment:
I think we should only subtract 1 when the height is nil (only when asked for the latest block). For example, if a user requests `Validators` for a specific height, you must not subtract 1.

I think it'll work as if `BlockResults()` were implemented.

Reply:
In this case, it still returns the validator set at the given height, the only difference is that instead it updates the light client to the height before it and uses the next validator hash from the light clients signed header to verify the validator set

Reply:
I can also change it back to updating the light client to the height provided and comparing with the (current) ValidatorHash as opposed to NextValidatorHash and then just have the exception that if the height is nil that we update the light client for the header before and then get the NextValidatorHash. I find both ways to be indifferent. Just remember that the latest validator set is always at a height one greater than the latest header